fiware-region-sanity-tests/resources/userdata.test.snat

```diff
@@ -0,0 +1,26 @@
+#cloud-config
+
+# phone_home: if this dictionary is present, then the phone_home
+# cloud-config module will post specified data back to the given
+# url
+# default: none
+# phone_home:
+#  url: http://my.foo.bar/$INSTANCE/
+#  post: all
+#  tries: 10
+#
+# Phone Home service will be listening in: fi-health.lab.fiware.org:8081
+phone_home:
+ url: http://fi-health.lab.fiware.org:8081/
+ post: [ hostname, instance_id ]
+ tries: 10
+
```

Comment:
I dislike putting explicitly here (GitHub) the Jenkins URL to be used. I'd prefer getting this information from elsewhere (maybe environment variables and then doing some text replacement?)